### PR TITLE
clean up only the temp dir, not the full work directory

### PR DESCRIPTION
**Pull request by natebrennand**
per https://github.com/Clever/sfncli/pull/9/files#r150304673

### CODE REVIEW

**Comment by mohit** [comment]
You might also want to remove this line: https://github.com/Clever/sfncli/blob/master/cmd/sfncli/sfncli.go#L72

**Comment by natebrennand** [comment]
Fixing the test now

File: cmd/sfncli/runner.go [hunk 1]
```diff
@@ -88,7 +90,7 @@ func (t *TaskRunner) Process(ctx context.Context, args []string, input string) e
 			return err
 		}
 
-		t.execCmd.Env = append(t.execCmd.Env, "WORK_DIR="+t.workDirectory)
+		t.execCmd.Env = append(t.execCmd.Env, fmt.Sprintf("WORK_DIR=%s", tmpDir))
```
Comment (mohit) [inline cmd/sfncli/runner.go]: :facepalm:

**Comment by natebrennand** [comment]
Side note, would be really nice if we didn't shadow `log`. Makes standard debugging painful when you can't pull in `log` automatically:

```diff
diff --git a/cmd/sfncli/runner_test.go b/cmd/sfncli/runner_test.go
index cd20e77..4533abf 100644
--- a/cmd/sfncli/runner_test.go
+++ b/cmd/sfncli/runner_test.go
@@ -3,6 +3,7 @@ package main
 import (
        "context"
        "encoding/json"
+       stdLog "log"
        "os"
        "path"
        "strings"
@@ -52,6 +53,7 @@ func (w *workdirMatcher) Matches(x interface{}) bool {
        }

        w.foundWorkdir = workdirBlob.Dir
+       stdLog.Println(workdirBlob.Dir, "vs", w.expectedPrefix, "from", *input.Output)
        return strings.HasPrefix(workdirBlob.Dir, w.expectedPrefix)
 }
```

**Comment by mohit** [comment]
@natebrennand I actually like it since it makes it hard to mistakenly use `stdLog` when you really should be using Kayvee in this repo. In any case let's leave that change for a different time. Can you give a look to https://github.com/Clever/sfncli/pull/10/commits/1f735ff7c520ac4ee2097e41b9c1c3d6de69a523

File: cmd/sfncli/sfncli.go [hunk 2]
```diff
@@ -70,7 +70,6 @@ func main() {
 			fmt.Println(err)
 			os.Exit(1)
 		}
-		defer os.RemoveAll(*workDirectory)
```
Comment (mohit) [inline cmd/sfncli/sfncli.go]: 🤦‍♂️ 🙇

File: cmd/sfncli/runner.go [hunk 3]
```diff
@@ -25,6 +25,7 @@ import (
 const maxTaskOutputLength = 32768
 const maxTaskFailureCauseLength = 32768
 
+// TaskRunner manages resources for executing a task
```
Comment (mohit) [inline cmd/sfncli/runner.go]: :DTEC:

**Comment by natebrennand** [comment]
> I actually like it since it makes it hard to mistakenly use stdLog when you really should be using Kayvee in this repo. In any case let's leave that change for a different time. Can you give a look to 1f735ff

Wasn't proposing the change, just commenting on the situation.
IMO that need would be more appropriately solved with a static check in Circle CI. Setting up a repo to violate default expectations, like being able to use any standard library package, is liable to slow down development.

**Pull request merged**